cmd/kops/rollingupdate_cluster.go

```diff
@@ -68,6 +70,7 @@ func init() {
 	}
 }
 
+// Run the Cobra Command - Do a rolling update for a Cluster
```

Comment:
If you're going to go to the effort of adding comments, you might as well keep golint happy.  golint is a fussy little fuss pot: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences